index.html

```diff
@@ -9,26 +9,32 @@
   </head>
 
   <body>
-    <label>
-      <input type="checkbox" id="filterConsent" />
-      Show only courses that require consent
-    </label>
+    <div class="filteritemsdisplay">
+      <label>
+        <input type="checkbox" id="filterconsent" />
+        Show only require consent courses
+      </label>
 
-    <form>
-      <label for="days">Choose a day:</label>
-      <select id="selecteddays">
-        <option value="M">Monday</option>
-        <option value="T">Tuesday</option>
-        <option value="W">Wednesday</option>
-        <option value="R">Thursday</option>
-        <option value="F">Friday</option>
-      </select>
-    </form>
+      <form>
+        <label for="days">Choose a day:</label>
```

Comment:
This label is for the element with id "days", but that id does not exist on the page. Presumably, `days` and `selecteddays` were intended to be the same identifier.